testall.sh

```diff
@@ -59,77 +59,78 @@ echo "----------------------------------------"
 #./target/release/zmu run tests/pi/pi-cm4f.elf
 
 
+# commenting these tests because they are not in the distribution
```

Comment:
good point, having these would require the script to checkout code from other git repo. Maybe future improvement would be separate testsuite setup step or interactive mode for downloading extra test code.

Reply:
are you familiar with submodules from git ? it's a way to do this. another approach is just to include the clone commands in the testall.sh. Some additional examples would be interesting, specially the coremark.

Reply:
Coremark is bit of a problem because of the licensing. Last time I checked they don't want it to be distributed otherwise than via their own website. But for other examples, git submodules is a great idea.